galpy/potential_src/SpiralArmsPotential.py

```diff
@@ -33,42 +33,60 @@ def func_wrapper(self, R, z, phi, t):
+
+    From (`Cox and Gomez 2002 <https://arxiv.org/abs/astro-ph/0207635>`__):
+     "In a particularly interesting example [of the choices for :math:`C_n`],
+     the density behaves approximately as a cosine squared in the arms but is separated by a flat interarm region
+     occupying half the volume."
+
+     It has three terms in its sum, with
+
+     .. math::
+        C_1 &= \\frac{8}{3 \\pi} \\\\
+        C_2 &= \\frac{1}{2} \\\\
+        C_3 &= \\frac{8}{15 \\pi} \\\\
     """
 
-    def __init__(self, amp=1, ro=None, vo=None, amp_units='density', normalize=False,
+    def __init__(self, amp=1, ro=None, vo=None, amp_units='density',
                  N=2, alpha=0.2, r_ref=1, phi_ref=0, Rs=0.3, H=0.125, omega=0, Cs=[1]):
+
         """
         NAME:       
             __init__
         PURPOSE:
-            initialize a spiral arm potential
+            initialize a spiral arms potential
         INPUT:
             :param amp: amplitude to be applied to the potential (default: 1); 
-                        can be a Quantity with units of density. (amp = 4 * pi * G *rho0)
-            :param normalize: if True, normalize such that vc(1.,0.)=1., or, if given as a number
-                              such that the force is this fraction of the force necessary to make vc(1.,0.)=1.
+                        can be a Quantity with units of density. (:math:`amp = 4 \\pi G \\rho_0`)
             :param ro: distance scales for translation into internal units (default from configuration file)
             :param vo: velocity scales for translation into internal units (default from configuration file)
             :param N: number of spiral arms
             :param alpha: pitch angle of the logarithmic spiral arms in radians (can be Quantity)
-            :param r_ref: fiducial radius where rho = rho0 (r_0 in the paper by Cox and Gomez) (can be Quantity)
-            :param phi_ref: reference angle (phi_p(r_0) in the paper by Cox and Gomez) (can be Quantity)
+            :param r_ref: fiducial radius where :math:`\\rho = \\rho_0` (:math:`r_0` in the paper by Cox and Gomez) (can be Quantity)
+            :param phi_ref: reference angle (:math:`\\phi_p(r_0)` in the paper by Cox and Gomez) (can be Quantity)
             :param Rs: radial scale length of the drop-off in density amplitude of the arms (can be Quantity)
             :param H: scale height of the stellar arm perturbation (can be Quantity)
-            :param Cs: list of constants multiplying the cos(n*gamma) term in the mass density expression
-            :param omega: rotational speed of the spiral arms (can be Quantity)
+            :param Cs: list of constants multiplying the :math:`cos(n \\gamma)` term in the mass density expression
+            :param omega: rotational pattern speed of the spiral arms (can be Quantity)
         OUTPUT:
             (none)
         HISTORY:
-            2017-05-12  Jack Hong (UBC)
+            2017-07-04  Jack Hong (UBC)
```

Comment:
Keep the full history here: date when you started it, and additional dates when you made big changes.